src/lib/utils/popups.ts

```diff
@@ -1,19 +1,30 @@
 import mapboxgl from 'mapbox-gl';
+import { getTranslation } from './translations';
+import { Languages } from '../types';
 
-export const createPopup = (feature: mapboxgl.MapboxGeoJSONFeature): mapboxgl.Popup => {
+export const createPopup = (
+    feature: mapboxgl.MapboxGeoJSONFeature,
+    lang: Languages
+): mapboxgl.Popup => {
     if (feature.geometry.type !== 'Point') {
         return null;
     }
 
     const [lon, lat] = feature.geometry.coordinates;
 
+    let stationName = feature.properties.name;
```

Comment:
You can rewrite this and the if below to
```js
const { name: stationName = getTranslation(lang, feature.layer.id) } = feature.properties
```
It means you get a `name` property out of `feature.properties` and name it `stationName` with a default value of `getTranslation(lang, feature.layer.id)`

Reply:
Cool. But I'm not sure I will be able to understand syntax like this ever again... :nerd_face: